django_rq/metrics_collector.py

```diff
@@ -0,0 +1,59 @@
+from rq.job import JobStatus
+
+from .queues import filter_connection_params, get_connection, get_queue, get_unique_connection_configs
+from .workers import get_worker_class
+
+try:
+    from prometheus_client import Summary
+    from prometheus_client.core import GaugeMetricFamily, CounterMetricFamily
+
+    class RQCollector:
```

Comment:
Do you mind moving this code to `contrib/prometheus.py`? Anything that is installed via `extras` should go in there.

Reply:
I didn't see that folder, but I created it.